README.md

```diff
@@ -57,6 +57,7 @@ options use-vc
 
 * [libkrunfw](https://github.com/containers/libkrunfw)
 * A working [Rust](https://www.rust-lang.org/) toolchain
+* C Library static libraries, as the [init](init/init.c) binary is statically linked.
```

Comment:
Perhaps it'd be helpful to put a couple of examples here (`glibc-static` in Fedora, whatever is called in Debian...).

Reply:
@slp agreed, it will be helpful. I will send a v2.